microsoft/testsuites/core/storage.py

```diff
@@ -0,0 +1,203 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import re
+from pathlib import PurePosixPath
+from typing import Optional
+
+from assertpy.assertpy import assert_that
+
+from lisa import (
+    LisaException,
+    Logger,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.environment import EnvironmentStatus
+from lisa.operating_system import CoreOs
+from lisa.tools import Cat, Mount
+from lisa.tools.lsblk import Lsblk
+from lisa.tools.swapon import SwapOn
+
+
+@TestSuiteMetadata(
+    area="storage",
+    category="functional",
+    description="""
+    This test suite is used to run storage related tests.
+    """,
+)
+class Storage(TestSuite):
+    DEFAULT_DISK_TIMEOUT = 300
+
+    # ResourceDisk.MountPoint=/mnt
+    _resource_disk_mount_point_regex = re.compile(
```

Comment:
They can be summarized to one pattern, extract key/value pairs, and check if the key is wanted.